lib/util/package-rules/repositories.spec.ts

```diff
@@ -1,11 +1,11 @@
 import { RepositoriesMatcher } from './repositories';
 
 describe('util/package-rules/repositories', () => {
-  const packageNameMatcher = new RepositoriesMatcher();
+  const repositoryMatcher = new RepositoriesMatcher();
```

Comment:
this rename makes a lot noise, we should do it in current main before

Reply:
https://github.com/renovatebot/renovate/pull/28652